client/src/App.js

```diff
@@ -13,16 +13,24 @@ export default class App extends Component {
   }
 
   addToSavedList = movie => {
+    //console.log(movie.title)
+    if (this.state.savedList.includes(movie)) {
+      alert(`${movie.title} is already in your list!`)
+    } else {
     const savedList = this.state.savedList;
```

Comment:
Nice use of the `.includes` array method, and it's great that you thought of preventing repeated saved movies!